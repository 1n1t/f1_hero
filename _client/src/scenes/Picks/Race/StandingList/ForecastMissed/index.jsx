import React from 'react';

const ForecastMissed = () => (
	<div className="standings-container">
		<div className="standing">
			You did not submit your prediction on time. No race results available for
			you.
		</div>
	</div>
);

export default ForecastMissed;
