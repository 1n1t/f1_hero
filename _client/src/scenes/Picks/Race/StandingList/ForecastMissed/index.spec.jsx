import r from 'Root/jest/utils/renderer';

import ForecastMissed from './index';

const defaultProps = {};

r.init(ForecastMissed, defaultProps);

it('renders correctly', () => {
	const wrapper = r.render();
	expect(wrapper).toMatchSnapshot();
});
