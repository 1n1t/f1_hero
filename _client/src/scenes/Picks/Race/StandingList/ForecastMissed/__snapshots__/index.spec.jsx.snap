// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="standings-container"
>
  <div
    className="standing"
  >
    You did not submit your prediction on time. No race results available for you.
  </div>
</div>
`;
